module2/projects/hang-in-there/index.md

```diff
@@ -0,0 +1,407 @@
+---
+title: Hang In There - Pair
```

Comment:
Should say Solo not Pair?

Reply:
Thank you!  Just pushed up a fix for that.